packages/octane-blueprint-utils/index.js

```diff
@@ -55,7 +55,7 @@ function getRepoVersion(org, repo) {
         return getRepoVersionFromTarball(org, repo);
       }
 
-      throw error;
```

Comment:
I'm surprised linting didn't catch this.

is linting even enabled on C.I.?!?

Reply:
Me too, and apparently not?